ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -110,7 +115,46 @@ export class Sharing extends React.Component<Props, State> {
         values: users.filter(e => e.access === Access.Share).map(e => e.value),
       },
     ]
+    this.attemptSave(attributes)
+  }
+
+  attemptSave = (attributes: any) => {
+    fetch('/search/catalog/internal/metacard/' + this.props.id)
+      .then(res => res.json())
+      .then(data => {
+        // Only allow a user to save if the previous workspace state is the same as the latest workspace state
```

Comment:
✏️ I don't really think this comment is all that necessary. I think the if statement is pretty self explanatory.

Reply:
Removed